src/VueSimpleIcon.ts

```diff
@@ -11,26 +11,28 @@ import { renderError } from "./util";
     if (!icon)
       return renderError("Icon not found", this.iconSize, createElement);
 
-    let svgContent = icon.svg.replace(/<\/?svg[^>]*>/g, "");
-    if (this.title) {
-      svgContent = svgContent.replace(
-        /<title>.*<\/title>/,
-        `<title>${this.title}</title>`
-      );
-    }
+    const title = this.title ? this.title : `${icon.title} icon`;
 
-    return createElement("svg", {
-      attrs: {
-        fill: this.color || `#${icon.hex}`,
-        width: this.iconSize,
-        height: this.iconSize,
-        viewBox: "0 0 24 24",
-        xmlns: "http://www.w3.org/2000/svg"
+    return createElement(
+      "svg",
+      {
+        attrs: {
+          fill: this.color || `#${icon.hex}`,
+          width: this.iconSize,
+          height: this.iconSize,
+          viewBox: "0 0 24 24",
+          xmlns: "http://www.w3.org/2000/svg"
+        }
       },
-      domProps: {
-        innerHTML: svgContent
-      }
-    });
+      [
+        createElement("title", title),
+        createElement("path", {
+          attrs: {
+            d: icon.path
```

Comment:
What's about multipath icons? have you tested them?

Reply:
The whole idea of the effort was to make sure all icons have only a single path, for more background see: https://github.com/simple-icons/simple-icons/issues/1275 and https://github.com/simple-icons/simple-icons/pull/1277.

Hence all icons have only one path and it's found as a string (see https://github.com/simple-icons/simple-icons/blob/b5c346cdd9ff4fe51dd83357d258c61f609837b5/tests/icons.test.js#L25) of the path property of the icon.